### PR TITLE
Add acknowledgements section and contributors

### PR DESCRIPTION
**Pull request by agitter**
This pull request adds an acknowledgements section for individuals to who helped us discuss the manuscript and issues but have not been included as an author.  I'll start by tagging people I directly interacted with.

If you are on the list below, please let me know how you would like your name to appear in the acknowledgements (or tell me you prefer to be left off):
- @rbharath
- @cangermueller
- @ueser
- @michaelmhoffman
- @hussius
- @gokceneraslan
- @kumardeep27
- @w9

### CODE REVIEW

**Comment by w9** [comment]
You could use my full name Xun Zhu. Thanks!

**Comment by kumardeep27** [comment]
You could use my full name Kumardeep Chaudhary. Thanks!

**Comment by gokceneraslan** [comment]
Gökcen Eraslan, thanks.

**Comment by hussius** [comment]
Mikael Huss. Cheers!

**Comment by agitter** [comment]
@cgreene I'll leave it open a while longer to keep adding names, then merge.

**Comment by rbharath** [comment]
Bharath Ramsundar, thanks.

**Comment by cangermueller** [comment]
Christof Angermueller, thanks!

File: sections/07_conclusions.md [hunk 1]
```diff
@@ -91,21 +91,25 @@ that deep learning has transformed biology and medicine.
...
+approved the final manuscript draft; and agreed to be accountable in all aspects
+of the work. Individuals who did not contribute in all of these ways, but who
+did participate, are acknowledged at the end of the manuscript.
 
 `TODO: update after finalizing discussion in #369`
+
+### Acknowledgements
+
+We gratefully acknowledge Christof Angermueller, Kumardeep Chaudhary, Gökcen
+Eraslan, Mikael Huss, Bharath Ramsundar and Xun Zhu
```
Comment (michaelmhoffman) [inline sections/07_conclusions.md]: Please add `Michael M.\ Hoffman`. Thanks!

**Comment by agitter** [comment]
@cgreene I added the other contributors who did not approve the final draft to the acknowledgements.  Some journals request that acknowledged individuals consent, so I used their GitHub ids.  We could switch to their full names.

**Comment by agitter** [comment]
I should have used a `##` header for the references section.  I'll fix it when I proofread the conclusions.

**Comment by rojinsafavi** [comment]
> You could use my full name Kumardeep Chaudhary. Thanks!

Dear Kumardeep, 
Would you please share the source code with use? thank you

**Comment by agitter** [comment]
Hello @rojinsafavi.  This GitHub repository was created for a collaborative review manuscript.  We haven't implemented any machine learning methods as part of this project.

**Comment by rojinsafavi** [comment]
Dear @agitter, Yes, I totally understand that this Github is for reviewing manuscripts. I think it is very important for authors to publish the code when they publish a paper, otherwise, no one can reproduce their results or test if it works on other datasets. And potentially the model cannot be compared to other models. It would be super helpful if reviewers take that into account :-)

I came across this discussion when I was looking for code: **Deep Learning–Based Multi-Omics Integration Robustly Predicts Survival in Liver Cancer**, and I had no success. Since none of the email addresses on the paper works, I was hoping that the author of the paper sees this comment and shares the code :-)

**Comment by evancofer** [comment]
@rojinsafavi To echo @agitter , that's understandable, but every comment here sends an email and notification to a large group of people watching the repository and this pull request. I doubt you intended this.

I have taken the liberty to look at the paper you mentioned and do a very quick Google search. According to the article, the principal investigator and corresponding author is Lana X. Garmire. You should be reaching out to her. She recently moved from the University of Hawaii at Manoa to the University of Michigan, so the email address on the publication is out of date. I would recommend using the email address that is on her [lab website](http://garmiregroup.org/). In the future, I would encourage you to first contact an editor if the corresponding author's email address is out-of-date. Doing so will help others avoid the difficulty you encountered.

**Pull request merged**